docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -44,10 +44,10 @@ Change directories to *FSNetCore* and start adding projects to the solution fold
 Use the `dotnet new` command, create a Class Library project in the **src** folder named Library. 
 
 ```bash
-dotnet new classlib -lang F# -o src/Library 
+dotnet new lib -lang F# -o src/Library 
```

Comment:
although this works because of the partial match, if someone goes to the dotnet new docs or type dotnet new, lib won't be an option. I'd prefer to keep the full short template name here.